pkg/util/paramtable/component_param.go

```diff
@@ -1445,6 +1445,13 @@ type queryNodeConfig struct {
 
 	// loader
 	IoPoolSize ParamItem `refreshable:"false"`
+
+	// schedule task policy.
+	SchedulePolicyName                    ParamItem `refreshable:"false"`
+	SchedulePolicyMaxPendingTask          ParamItem `refreshable:"true"`
```

Comment:
how this works diff to MaxReadConcurrency, should it be always not greater than MaxReadConcurrency?

Reply:
I think that `MaxReadConcurrency` is no comparison relationship with `SchedulePolicyMaxPendingTask`.

- `MaxReadConcurrency` limits the max concurrent running tasks.
- `SchedulePolicyMaxPendingTask` limits the max pending tasks. query scheduler will reject new incoming task if the pending number of task exceeds this parameter. If the pending task queue is too long, the scheduler will cost much time to handle merging logic. It is just a protective logic that will not trigger in normal usage mode.